.github/workflows/load-testing.yml

```
@@ -0,0 +1,294 @@
+          # Pull required models for load testing
+          ollama pull qwen2.5:0.5b
+          ollama pull nomic-embed-text
+          
+      - name: Install k6
+        run: |
+          sudo gpg -k
+          sudo gpg --no-default-keyring --keyring /usr/share/keyrings/k6-archive-keyring.gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys C5AD17C747E3415A3642D57D77C6C491D6AC1D69
+          echo "deb [signed-by=/usr/share/keyrings/k6-archive-keyring.gpg] https://dl.k6.io/deb stable main" | sudo tee /etc/apt/sources.list.d/k6.list
+          sudo apt-get update
+          sudo apt-get install k6
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)